app/controllers/dashboards_controller.rb

```diff
@@ -21,7 +21,7 @@ def show
     fetch_and_authorize_user
     target = @user
     # NOTE: This is a subtle policy check happening here that we are not encapsulating
-    not_authorized if params[:org_id] && !@user.org_admin?(params[:org_id] || @user.any_admin?)
```

Comment:
how did this ever work before?
`user.org_admin?(params[:org_id] || @user.any_admin?)`